mealie/pkgs/safehttp/transport.py

```diff
@@ -46,8 +46,6 @@ async def handle_async_request(self, request):
         netloc = request.url.netloc.decode()
         if ":" in netloc:  # Either an IP, or a hostname:port combo
             netloc_parts = netloc.split(":")
-            if len(netloc_parts) > 1:  # netloc of username:password@hostname:port not supported
```

Comment:
@hay-kot see this log.. showing that even when `user:pass` is included in the url, it was being stripped out before getting to this netloc variable. Thus, the check on parts is irrelevant here.

![image](https://github.com/mealie-recipes/mealie/assets/3479092/49f3101b-c775-4eb4-a534-cbcfca795f9e)